x-pack/plugins/event_log/server/plugin.test.ts

```diff
@@ -42,7 +44,7 @@ describe('event_log plugin', () => {
     const plugin = new Plugin(initializerContext);
     const spaces = spacesMock.createStart();
     // serverless setup is currently empty, and there is no mock
-    plugin.setup(coreSetup, { serverless: {} });
+    plugin.setup(coreSetup, { serverless: { setupProjectSettings(keys: string[]) {} } });
```

Comment:
Seems like we should use the mock here, so we don't have to change this code when the interface changes.